hgtv/templates/index.html.jinja2

```diff
@@ -1,101 +1,18 @@
-            <div class="video-box__video">
-              <iframe id="iframe-stream2" src={{config['LIVESTREAM_2']}} frameborder="0" allowfullscreen></iframe>
-            </div>
-          </div>
-        {% endif %}
-      </div>
-    </div>
-  {% endif %}
-  <ul class="grid">
-    {%- for channel in channels -%}
-      <li class="grid__col-12 grid__col-xs-12 grid__col-sm-6 grid__col-lg-4">
-        <div class="grid__cell">
-          <a href="{{ channel.url_for() }}" class="clickable-card">
-            <div class="card">
-              <div class="card__image-wrapper">
-                {% if channel.channel_banner_url %}
-                  <img class="card__image" src="{{ channel.channel_banner_url }}" alt="{{ channel.title }}"/>
-                {% else %}
-                  <div class="card__image--default">
-                    <img class="card__image" src="https://images.hasgeek.com/embed/file/5084bed3b3254c548bdcc119cd7b12ef" alt="{{ channel.title }}"/>
-                    <p class="card__image__tagline">{{ channel.title }}</p>
-                  </div>
-                {% endif %}
-                <div class="card__logo">
-                  <div>
-                  {% if channel.channel_logo_filename %}
-                    {{ render_thumbnail(channel.channel_logo_filename, class='card__logo__img', alt=channel.title) }}
-                  {% else %}
-                    <img class="card__logo__img" src="{{ url_for('static', filename='img/sample-logo.png') }}" alt="{{ channel.title }}"/>
-                  {% endif %}
-                  </div>
-                </div>
-              </div>
-              <div class="card__title">
-                <h3 class="mui--text-title card__title__heading">{{ channel.title }}</h3>
-              </div>
-              {% if channel.bio %}
-                <div class="card__body">
-                  <p>{{ channel.bio }}</p>
-                </div>
-              {% endif %}
-              <div class="mui-divider"></div>
-              <div class="card__footer">
-                <a href="{{ channel.url_for() }}" class="mui-btn mui-btn--flat mui-btn--accent card__footer__floatbtn">
-                  <i class="material-icons play-btn">play_circle_filled</i>
-                </a>
-              </div>
-            </div>
-          </a>
-        </div>
-      </li>
-    {% else %}
-      <p>No Channels available</p>
-    {%- endfor -%}
-  </ul>
+{% extends "home.html.jinja2" %}
+{%- block layoutheaders %}
+	
+	  <link href="/static/js/dist/css/app.a92fcd107bae02bc79c0b00cce49b4b0.css" rel="stylesheet">
+	
 {% endblock %}
 
-{% block footerscripts %}
-<script src="//platform.twitter.com/widgets.js" type="text/javascript"></script>
-<script type="text/javascript">
-  $(document).ready(function(){
-    $('#tab-container').easytabs();
-  });
-</script>
-{% endblock %}
+{%- block appcontent %}
+	<div id="app"></div>
 
+	
+	<script src="/static/js/dist/js/manifest.951e00270cd951bf20e5.js"></script>
```

Comment:
These numbers shouldn't be hard-coded, no? They'll have to be updated every time there's a change.

Reply:
This template is generated by webpack everytime we build js.